connectionchecker/connection_checker.go

```diff
@@ -206,3 +210,26 @@ func (cc *ConnectionChecker) checkProxySQLConnection(ctx context.Context, dsn st
 
 	return &res
 }
+
+func (cc *ConnectionChecker) checkExternalConnection(ctx context.Context, uri string) *agentpb.CheckConnectionResponse {
+	var res agentpb.CheckConnectionResponse
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
let's just use context coming in args.
As far as I see we pass timeout here https://github.com/percona/pmm-agent/pull/231/files#diff-6eb9fa169fdb33f5cf7bf4a8267eef04668d3cee3ccb099593589a30067c141aR63